src/components/Common/Misc/ContentSelector/ContentSelector.js

```diff
@@ -3,41 +3,17 @@
  * Agent Holder, and Agent Verifier pages based on value of persona
  * @returns The ContentWrap component.
  */
-import ContentWrap01 from '../../../AgentIssuer/ContentWrap'
-import ContentWrap02 from '../../../AgentHolder/ContentWrap'
-import ContentWrap03 from '../../../AgentVerifier/ContentWrap'
+import ContentWrap from '../../../AgentAuthority/ContentWrap'
 
 export default function ContentSelector({ children, status, origin, persona }) {
```

Comment:
is `persona` still used anywhere?

Reply:
yes, `persona` in this context comes from `GET /status` on the cloudagent (`label` in the response) . The call to `/status` happens when the agent endpoint is set by user after first loading the home page. That `persona` gets used for all the p2p connection stuff and when cred defs are created. We could change the variable name? (`alias` seems to match what the API spec calls it). Any thoughts @dc-andysign?

Reply:
hang on but why do we have this? Unless I'm missing something `persona` only gets passed to `ContentWrap` which pases it to `ContentLeftWrap` which then doesn't use it. I guess the question is why do we get the persona from `GET /status`? Where in the app is this used?

Reply:
oh right sorry yes it's not used in `ContentWrap` since this PR empties that out, I left it in there because I presumed whatever would be put in would probably use it, but I can remove it.

my previous comment was more about how `persona` is used generally, the only place it's actually used in this repo is within `ConnectivityWrap`

Reply:
I've removed persona entirely, and the use of aliases when creating/receiving invitations

Reply:
👍